packages/server/test/unit/request_spec.coffee

```diff
@@ -411,14 +411,14 @@ describe "lib/request", ->
             "Request Headers":  {"accept": "*/*", "accept-encoding": "gzip, deflate", "connection": "keep-alive", "host": "www.github.com"}
             "Request URL":      "http://www.github.com/dashboard"
             "Response Body":    null
-            "Response Headers": {"location": "/auth"}
+            "Response Headers": {"content-type": "application/json", "location": "/auth"}
```

Comment:
this change does not look intended...

Reply:
when nock upgraded, this was also updated, i assume they've started setting a default "content-type" for json responses since 9.x

Reply:
since it's isolated to the server's `nock` update, it only affects the unit tests, so shouldn't be an issue